compiler/damlc/tests/src/DA/Test/ScriptService.hs

```diff
@@ -101,7 +101,7 @@ main =
                       "      do pure v",
                       "testCreate = do",
                       "  p <- allocateParty \"p\"",
-                      "  submit p $ createCmd (T p 42)",
+                      "  submit p $ createCmd (T p 42)" ,
```

Comment:
?

Reply:
8c67d2a28f5f97c105f731301f7072bd1c4cc0d8 not sure how that snuck in